src/views/mainLayout.js

```diff
@@ -13,6 +14,7 @@ const mainLayout = data => {
   const metaTitle = data.meta?.socialTitle ?? getMessage('brand-fx-monitor')
   const metaDescription = data.meta?.socialDescription ?? getMessage('meta-desc-2')
   const pageUrl = `${AppConstants.SERVER_URL}${data.pathname ?? '/'}`
+  const pageHasFloatingBanner = AppConstants.FLOATING_BANNER_PAGES?.split(',').includes(data.pathname)
```

Comment:
We used this snippet above on  src/views/guestLayout.js#L17. Should we extract it into some shared helper function in case we tweak the banner logic in the future?

Reply:
Just moved the logic to determine on which pages to show the banner into the partial it self: [6692541](https://github.com/mozilla/blurts-server/pull/3067/commits/66925414402b07e8a3bc9fb4b75396f1d6a78b6e).